app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::CommentsController < ApplicationController
+  load_and_authorize_resource
+  def index
+    @comments = Comment.all
+    render json: @comments
+  end
```

Comment:
- [ ] Please make sure when I try to access comments of a specific post like for example `/users/1/posts/1/comments` API endpoints only lists the comments of post 1 by user 1 and not all the comments in the database.
- [ ] [OPTIONAL] Please fix the `initialize` method in the ability.rb model as its causing an error which is hindering the smooth running of the API testing